static/app/views/starfish/components/slideOverPanel.tsx

```diff
@@ -4,30 +4,41 @@ import styled from '@emotion/styled';
 import {motion} from 'framer-motion';
 
 const PANEL_WIDTH = '50vw';
+const PANEL_HEIGHT = '50vh';
 
 type SlideOverPanelProps = {
   children: React.ReactNode;
   collapsed: boolean;
   onOpen?: () => void;
+  slideDirection?: 'Right' | 'Bottom';
```

Comment:
Also a nit, but in TypeScript it's less common to capitalize this way. This should be either `'right' | 'bottom'` or you can declare an enum called `Position` and do `: Position` as the type